src/ci/scripts/dump-environment.sh

```diff
@@ -17,3 +17,9 @@ set +o pipefail
 du . | sort -nr | head -n100
 set -o pipefail
 echo
+
+echo "biggest files:"
+set +o pipefail
+du -h / 2>/dev/null | sort -hr | head -n100
+set -o pipefail
+echo
```

Comment:
It takes quite a long time on Windows environment so will cause timeout ([log](https://dev.azure.com/rust-lang/rust/_build/results?buildId=21231&view=logs&j=a6a2dd6f-5140-5fa9-b933-dfe57be2bdd5&t=5e47d2f0-69a7-59e9-0625-dc2d6fb92691)). I believe it's for debugging so can be removed now?

Reply:
Removed that piece of code.

Reply:
Unfortunate, but okay. I guess it makes sense it would be slow on Windows.